_rules/device-motion-user-interface-7677a9.md

```diff
@@ -26,19 +26,21 @@ htmlHintIgnore:
 
 ## Applicability
 
-The rule applies to an [HTML document][] with an associated [Window object][] that has an [event listener list][] with one or more [event listeners][event listener] for [device orientation events][device orientation] or [device motion events][device motion].
+The rule applies to any [HTML document][] with an associated [Window object][] that has an [event listener list][] with one or more [event listeners][event listener] for [device orientation events][device orientation] or [device motion events][device motion].
 
 ## Expectation 1
 
-For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target, a set of one or more [instruments][instrument] is available supporting the same outcomes of the event.
+For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target, a set of one or more [instruments][instrument] is available in the same [web page][] supporting outcomes indistinguishable of those from the event.
```

Comment:
If the instrument isn't visible at all times, it sounds like it would fail this rule. That doesn't seem like it should fail the SC though. Especially on mobile apps, it's pretty common for functionality to be organised in menus / popups / collapsable groups. Doesn't seem like that should fail the rule.

Reply:
Removed the reference to being available in the same page. Replaced it with the same solution that I used here: https://github.com/act-rules/act-rules.github.io/pull/960#discussion_r423577207